src/client/utils/validations.js

```diff
@@ -239,8 +239,15 @@ function isValidVeteranAddress(data) {
 }
 
 function isValidContactInformationSection(data) {
+  let emailConfirmationValid = true;
+
+  if (isNotBlank(data.email.value) && isBlank(data.emailConfirmation.value)) {
+    emailConfirmationValid = false;
+  }
```

Comment:
Yep, good catch! Just added that in a 2nd commit, forgot to include this ticket in the PR: https://github.com/department-of-veterans-affairs/healthcare-application-team/issues/407